modules/misc/doc/content/modules/misc/sqa/misc_sdd.md

```diff
@@ -1 +1,26 @@
 !template load file=sqa/module_sdd.md.template category=misc module=Misc
+!template! item key=introduction
```

Comment:
```suggestion

!template! item key=introduction
```

This intro isn't being registered in the rendered site. I think we need to add a space between the `template load` and `template item` commands.